week-6/Homework/mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,4 +1,39 @@
-function setAlarm() {}
+function setAlarm() {
+  let count = document.getElementById("timeRemaining");
+  let setIt = document.getElementById("set");
+  let pause = document.getElementById("p");
+  let Restart = document.getElementById("re");
+  let Counter;
```

Comment:
The naming of the variables names should be consistent. I'd suggest following the standard naming conventions, for variable names the naming convention is usually camel case that means that the first-word start is traight lowercased and then the first letter of each following words are uppercased.
 
Check this article out:
https://www.freecodecamp.org/news/javascript-naming-conventions-dos-and-don-ts-99c0e2fdd78a/